sktime/classification/compose/tests/test_feature_importances_.py

```diff
@@ -0,0 +1,85 @@
+
+import numpy as np
+import pytest
+from sklearn.base import clone
+from sklearn.pipeline import Pipeline
+from sklearn.pipeline import FeatureUnion
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.preprocessing import FunctionTransformer
+from sktime.datasets import load_gunpoint
+from sktime.utils.time_series import time_series_slope
+from sktime.transformers.series_as_features.segment import \
+    IntervalSegmenter
+from sktime.transformers.series_as_features.compose import \
+    RowTransformer
+from sktime.transformers.series_as_features.summarize._extract import \
+    RandomIntervalFeatureExtractor
+from sktime.classification.compose._ensemble import TimeSeriesForestClassifier
+
+
+# Compare simple cases of feature importances results obtained
+# directly from TimeSeriesForestClassifier with pipeline implementation
+# on the same intervals using the same DecisionTreeClassifier
+
+@pytest.mark.parametrize("n_intervals", [1, 2])
+@pytest.mark.parametrize("n_estimators", [1, 2])
+def test_feature_importances_(n_intervals, n_estimators):
+
+    random_state = 1234
+    n_features = 3
+    X_train, y_train = load_gunpoint(split="TRAIN", return_X_y=True)
+
+    # Compute feature importances using the default method
+    features = [np.mean, np.std, time_series_slope]
```

Comment:
the default method should now work without passing anything to estimator, so we can just call 
```python
TimeSeriesForestClassifier(random_state=random_state, n_estimators=n_estimators)
```

Reply:
An estimator was passed so that n_intervals could be specified each time (1 or 2 in this case). Would this still be achieved through calling TimeSeriesForestClassifier without passing anything to estimator?